src/compiler/emitter.ts

```diff
@@ -4707,6 +4707,9 @@ if (typeof __param !== "function") __param = function (paramIndex, decorator) {
                 }
 
                 write("[\"require\", \"exports\"");
+                if (compilerOptions.noEmitExtends) {

```

Comment:
Because this is targeting amd modules, as stated in #1350 the following module could be defined.

```js
define('__extends', function () {
  return function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
     __.prototype = b.prototype;
    d.prototype = new __();
  };
});
```